includes/business-hours-functions.php

```diff
@@ -333,6 +404,14 @@ function geodir_parse_hours_range( $hours_str ) {
 	return $return;
 }
 
+/**
+ * Get Business hours.
```

Comment:
for what, i assume this is used for frontend out put display, need better descriptions